forge-gui/res/cardsfolder/upcoming/incinerator_of_the_guilty.txt

```diff
@@ -0,0 +1,12 @@
+Name:Incinerator of the Guilty
+ManaCost:4 R R
+Types:Creature Dragon
+PT:6/6
+K:Flying
+K:Trample
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigImmediateTrig | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you may collect evidence X. When you do, CARDNAME deals X damage to each creature and each planeswalker that player controls. (To collect evidence X, exile cards with total mana value X or greater from your graveyard.)
+SVar:TrigImmediateTrig:AB$ ImmediateTrigger | Cost$ CollectEvidence<X> | Execute$ TrigDamage | TriggerDescription$ When you do, CARDNAME deals X damage to each creature and each planeswalker that player controls. (To collect evidence X, exile cards with total mana value X or greater from your graveyard.)
```

Comment:
can you pay 0?
as usual you'll need to remember the amount for the trigger

Reply:
Hmm i'm having difficulty having it remember the amount paid

Reply:
that should be fine
your problem is probably with the valid since you're in a different trigger context
add "Spawner>" syntax

Reply:
Ok it's working now, though I can still pay 0, is that correct ?

Reply:
yes see release notes